src/models/tools/table/table-content.ts

```diff
@@ -26,14 +26,13 @@ export function defaultTableContent() {
 }
 
 export function isLinkableValue(value: number | string | undefined) {
-  if ((value == null) || (value === "")) return false;
-  return isFinite(Number(value));
+  return value == null || Number.isNaN(value as number) || isFinite(Number(value));
```

Comment:
```suggestion
  return value == null || Number.isNaN(value as any) || isFinite(Number(value));
```
Given that the reason for using `Number.isNaN()` instead of global `isNaN()` is because of its handling of strings that don't convert to numbers, it seems counter-intuitive to use a TypeScript cast to claim that we believe the value is already a number. In fact, I'm inclined to think that the TypeScript typing for the `Number.isNaN()` function is incorrect, given that its return value is well-defined for any JavaScript value. I'm not thrilled about the cast to `any`, but it seems less disingenuous than the alternatives.

Reply:
I had similar thoughts - agreed this typing is more honest